linera-service/src/linera/main.rs

```diff
@@ -1235,48 +1235,6 @@ impl Runnable for Job {
                 );
             }
 
-            Wallet(WalletCommand::Init {
```

Comment:
Why is it safe to remove handling of this command here now?

Reply:
We `match` on the command in two places:
* In the `run()` function, where storage is not yet initialized. `Wallet::Init` is handled there, because as part of wallet initialization we also initialize storage.
* In the `<Job as Runnable>::run<S>(self, storage: S)` method, where storage is already available. We used to assign and sync the new chain there, but now that's not necessary anymore.